### DISCUSSION TITLE
How to remove the startup logo?

### DISCUSSION THREAD

**Discussion by mayuanfei** [hazelcast/hazelcast#25967] [General]
In a Spring Boot project using Hazelcast in an embedded mode, how can I remove the Hazelcast startup logo?

**Comment by rodrigoserracoelho** [comment]
Hello, did you manage to disable the logo/banner?

**Comment by kicktipp** [comment]
Add this line to `application.properties`

```
logging.level.com.hazelcast.system.logo=OFF
```

**Reply by rodrigoserracoelho** [reply]
It worked, thanks a lot!
